File: Backend/licorstore/src/main/java/com/aglayatech/licorstore/model/Estado.java
package com.aglayatech.licorstore.model;

import java.io.Serializable;

public class Estado implements Serializable {
	
	private Integer idEstado;
	private String estado;
	
	public Estado() {
		
	}

	public Integer getIdEstado() {
		return idEstado;
	}

	public void setIdEstado(Integer idEstado) {
		this.idEstado = idEstado;
	}

	public String getEstado() {
		return estado;
	}

	public void setEstado(String estado) {
		this.estado = estado;
	}
	
	private static final long serialVersionUID = 1L;

}

File: Frontend/licor-frontend/src/app/models/estado.ts
export class Estado {
    private idEstado: number;
    private estado: string;

    constructor(){}

    public getIdEstado(): number{
        return this.idEstado;
    }
    // tslint:disable-next-line: typedef
    public setIdEstado(idEstado: number){
        this.idEstado = idEstado;
    }

    public getEstado(): string{
        return this.estado;
    }
    // tslint:disable-next-line: typedef
    public setEstado(estado: string){
        this.estado = estado;
    }
}
